Firebase.Database/source/Firebase.Database/ApiDefinition.cs

```diff
@@ -17,6 +18,16 @@ interface Database
 		[Export ("database")]
 		Database DefaultInstance { get; }
 
+		// + (FIRDatabase *)databaseWithURL:(NSString *)url NS_SWIFT_NAME(database(url:));
+		[Static]
+		[Export ("databaseWithURL:")]
+		Database From (string url);
+
+		// + (FIRDatabase *)databaseForApp:(FIRApp *)app URL:(NSString*) url NS_SWIFT_NAME (database(app:url:));
+		[Static]
+		[Export ("databaseForApp:URL:")]
+		Database From (Firebase.Core.App app, string url);
```

Comment:
Consider `Create` instead of `From`

Reply:
I have another method with `From`, do you want me to change the other one and create an Obsolete method with `From`?